FogCreek/Glitch-Community
# boundless-raccoon-1 letting super_users edit project names/descriptions directly

## Links
* https://boundless-raccoon-1.glitch.me/
* https://glitch.manuscript.com/f/cases/3326627/

## Changes:
* makes project page editable for super_users

## How To Test:
1. with a super_user account try to edit a project name/description that does not belong to you. Here is a test account with test project I don't care about: https://boundless-raccoon-1.glitch.me/~faithful-thistle
2. verify you can see that action in the logs along with who did that action. this can be done with the glitchdb command, for those not that great with sql queries (such as myself) we talked about how to do this here: https://fogcreek.slack.com/archives/CGFKN6HSS/p1556047277027700
3. verify that once super_user is turned off you don't have this super power
4. verify that you can still edit your own stuff fine. 

## Feedback I'm looking for:
* anything I'm missing? security/privacy concerns I didn't think about?

## Things left to do before deploying:
- [ ] test this again once I have super powers or get someone else with super powers to test
